src/index.html

```diff
@@ -21,7 +24,482 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__picture"
+        href="./images/logo.png"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO-logo"
+        />
+      </a>
+      <nav>
+        <ul class="header__navlist">
+          <li class="header__navlist-items">
+            <a
+              href="#"
+              class="is-active header__navlist-linkstyle"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__navlist-items">
+            <a
+              class="header__navlist-linkstyle"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
The nav element should have a role='navigation' attribute for better accessibility.